src/libraries/Microsoft.PowerFx.Core/Lexer/Tokens/CommentToken.cs

```diff
@@ -11,13 +11,14 @@ namespace Microsoft.PowerFx.Syntax
     /// </summary>
     public sealed class CommentToken : Token
     {
-        internal bool IsOpenBlock;
+        internal readonly bool IsOpenBlock;
 
-        internal CommentToken(string val, Span span)
+        internal CommentToken(string val, Span span, bool isOpenBlock = false)
```

Comment:
>internal [](http://example.com/codeflow?start=8&length=8)

it's possible this might be internal breaking on PA; 
that's ok - trivial fix , just fyi.